source/2020-06-05-the-ember-times-issue-151.md

```diff
@@ -17,14 +18,21 @@ READMORE
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [`ember new --lang` RFC 🌐](https://github.com/emberjs/rfcs/pull/635)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+There is an [RFC](https://github.com/emberjs/rfcs/pull/635) to introduce the `--lang` flag to Ember CLI and help brand new Ember apps meet a [Level A guideline for WCAG](https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html). (Level A is the minimum level of conformance.) If the flag is used with a valid language code, the CLI will assign the `lang` attribute to the root `<html>` element in `app/index.html`.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+How will the `lang` attribute help your end users?
+
+> "Both assistive technologies and conventional user agents can render text more accurately when the language of the Web page is identified. Screen readers can load the correct pronunciation rules. Visual browsers can display characters and scripts correctly. Media players can show captions correctly. As a result, users with disabilities will be better able to understand the content."
+>
+> [WCAG Success Criterion 3.1.1: Intent](https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html#intent)
+
+<!-- alex ignore just -->
+Please note that _everyone_ benefits from having a valid page language specified, not just users with disabilities. To learn more about the importance of `lang` attribute and provide feedback, we encourage you to [check out the RFC](https://github.com/emberjs/rfcs/pull/635).
```

Comment:
```suggestion
Please note that <span style="font-style: italic;">everyone</span> benefits from having a valid page language specified, not just users with disabilities. To learn more about the importance of `lang` attribute and provide feedback, we encourage you to [check out the RFC](https://github.com/emberjs/rfcs/pull/635).
```

The repo actually doesn't honor `_` in markdown. But I didn't investigate a way to fix yet: https://github.com/ember-learn/ember-blog/issues/568